e2e/test/scenarios/native-filters/sql-filters-source.cy.spec.js

```diff
@@ -153,7 +153,23 @@ describe("scenarios > filters > sql filters > values source", () => {
       cy.createQuestion(structuredSourceQuestion).then(
         ({ body: { id: sourceQuestionId } }) => {
           cy.createNativeQuestion(
-            getStructuredTargetQuestion(sourceQuestionId),
+            getStructuredDimensionTargetQuestion(sourceQuestionId),
+          ).then(({ body: { id: targetQuestionId } }) => {
+            visitEmbeddedPage(getQuestionResource(targetQuestionId));
+          });
+        },
+      );
+
+      FieldFilter.openEntryForm();
+      checkFilterValueNotInList("Doohickey");
+      FieldFilter.selectFilterValueFromList("Gizmo");
+    });
+
+    it("should be able to use a structured question source when embedded with a text tag", () => {
```

Comment:
This diff is hard to read, but the change is just about adding a few more e2e tests to ensure custom parameter sources work with text tags (not field filters) in embedded and public contexts for different source types.